### DISCUSSION TITLE
how to modify body with response-rewrite plugin

### DISCUSSION THREAD

**Discussion by wkharrat** [apache/apisix#7101] [Q&A]
i need to parse json from backend and modify it, for example remove urls for pagination, the consumer don't want it. And i can't modify this in backend,

**Comment by tokers** [comment]
See https://apisix.apache.org/zh/docs/apisix/next/plugins/serverless/. Try to set the phase to `body_filter`.

**Reply by wkharrat** [reply]
Thanks for the suggestion, how to debug or get errors stack, i run apisix from docker compose, and not knowing lua, add this script in the plugin : 
`{
  "disable": false,
  "functions": [
    "return function(conf, ctx)  ngx.arg[1] = \"test\"..ngx.arg[1] end"
  ],
  "phase": "rewrite"
}`

**Reply by tokers** [reply]
> 

See this sentence carefully. I mean the phase is `body_filter`.
